test/unit/BeefyZapBiswapLPVaultOperator.ts

```diff
@@ -173,13 +215,9 @@ describeOnBscFork("BeefyZapBiswapLPVaultOperator", () => {
             // Moo tokens in vault
             const mooBalanceReserve = await vault.balanceOf(context.nestedReserve.address);
             expect(mooBalanceReserve).to.not.be.equal(BIG_NUMBER_ZERO);
-
             expect(await context.user1.getBalance()).to.be.equal(ethBalanceBefore.sub(bnbToDepositAndFees).sub(txFees));
 
-            /*
-             * I can't predict the WBNB received in the FeeSplitter.
-             * It should be greater than 0.01 WBNB, but sub 1% to allow some dust sent back to the user (without fees)
-             */
+            // Fees in ETH
             expect(await context.WBNB.balanceOf(context.feeSplitter.address)).to.be.gt(
```

Comment:
If we can now predict the WBNB received in the FeeSplitter, use `expect().to.eq()` instead of `expect().to.be.gt()`.

Reply:
Indeed !
Should be better with  new commits